flocker/node/functional/retry-docker/Dockerfile

```diff
@@ -2,6 +2,10 @@
 # in /data/count.
 FROM python:2.7-slim
 VOLUME /data
-ADD run /usr/local/bin/run
 WORKDIR /data
+
+ADD run /usr/local/bin/run
```

Comment:
For the record, this line was not added, just moved below the WORKDIR command.  The actual change is explained in the subsequent comment.